### PR TITLE
[New Package] Perl YAML::Tiny

### PR DESCRIPTION
**Pull request by saltedcoffii**

### CODE REVIEW

File: packages/perl_yaml_tiny.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  description 'YAML::Tiny - Read/Write YAML files with as little code as possible'
+  homepage 'https://metacpan.org/pod/YAML::Tiny'
+  version '1.63'
+  compatibility 'all'
+  source_url 'https://cpan.metacpan.org/authors/id/E/ET/ETHER/YAML-Tiny-1.73.tar.gz'
+  source_sha256 'bc315fa12e8f1e3ee5e2f430d90b708a5dc7e47c867dba8dce3a6b8fbe257744'
+
+  depends_on 'perl'
+
+  def self.build
```
Comment (supechicken) [inline packages/perl_yaml_tiny.rb:13]: Same as `postinstall` and `check` part, if nothing need to be done in this part, just remove `def self.build` is fine

File: packages/perl_yaml_tiny.rb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  end
+
+  def self.install
+    # install files to build directory
+    system 'cpanm', '-l', 'build', '--self-contained', '--force', '.'
+
+    # install lib
+    libdir = CREW_DEST_DIR + `perl -e 'require Config; print $Config::Config{'"'installsitelib'"'};'`
+    FileUtils.mkdir_p libdir
+    system "(cd build/lib/perl5; tar cf - .) | (cd #{libdir}; tar xfp -)"
```
Comment (supechicken) [inline packages/perl_yaml_tiny.rb:23]: Rubyize with `FileUtils.cp_r Dir.glob('build/lib/perl5/*'), libdir`?

File: packages/perl_yaml_tiny.rb [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+    # install man
+    FileUtils.mkdir_p CREW_DEST_MAN_PREFIX
+    system "(cd build/man; tar cf - .) | (cd #{CREW_DEST_MAN_PREFIX}; tar xfp -)"
```
Comment (supechicken) [inline packages/perl_yaml_tiny.rb:27]: Rubyize with FileUtils.cp_r Dir.glob('build/man/*'), CREW_DEST_PREFIX`?

File: packages/perl_yaml_tiny.rb [hunk 4]
```diff
@@ -0,0 +1,32 @@
+require 'package'
+
+class Perl_yaml_tiny < Package
+  description 'YAML::Tiny - Read/Write YAML files with as little code as possible'
+  homepage 'https://metacpan.org/pod/YAML::Tiny'
+  version '1.63'
```
Comment (supechicken) [inline packages/perl_yaml_tiny.rb:6]: Do you mean `version '1.73'` ?

**Comment by saltedcoffii** [comment]
Alright those changes should be good

**Pull request merged**